app/assets/stylesheets/components/_markdown-editor.scss

```diff
@@ -77,6 +77,8 @@ $toolbar-button-size: 50px;
 
   .govuk-textarea {
     overflow: scroll;
+    padding: 30px;
+    background-color: govuk-colour("grey-4");    
```

Comment:
We need to remove the trailing empty spaces at the end of this line to get the CI/linter pass.

Reply:
Done